README.md

```diff
@@ -1,17 +1,32 @@
 # Mapbox Annotation Extension for iOS
 
-The Mapbox Annotation Extension is lightweight library you can use with the Mapbox Maps SDK to quickly add basic shapes to a map.
+The Mapbox Annotation Extension is a lightweight library you can use with the Mapbox Maps SDK for iOS to quickly add basic shapes to a map.
 
-⚠️ This product is currently in active alpha development, and is not intended for usage in production. We welcome feedback
-from developers in issues. ⚠️
+This extension leverages the power of the Mapbox runtime styling api with an object oriented approach to simplify the creation and styling of annotations.
+
+⚠️ This product is currently in active beta development, and is not suitable for production usage. ⚠️
 
 ## Installation
 
-### Download and unzip the latest release
+### Using CocoaPods
+To install Mapbox Annotation Extension using CocoaPods:
+
+Create a Podfile with the following specification:
+
+pod 'MapboxAnnotationExtension', '0.0.1-beta.1'
+Run pod repo update && pod install and open the resulting Xcode workspace.
+
+### Using Carthage
+Alternatively, to install Mapbox Navigation using Carthage:
+
+Create a Cartfile with the following dependency:
+
+github "mapbox/mapbox-annotation-extension" ~> 0.0.1-beta.1
+Run carthage update --platform iOS to build just the iOS dependencies.
 
-Reach out to us to get a copy of the binaries [through our contact page](https://support.mapbox.com/).
+### Manual
 
-### Embed the framework in your project
+Download and unzip the latest [release](https://github.com/mapbox/mapbox-annotation-extension/releases).
 
 Drag **MapboxAnnotationExtension.framework** into your project's Embedded Binaries section in the project editor. In the sheet that appears, make sure "Copy items if needed" is checked, then select Finish.
 
```

Comment:
Can you add a section after installation about the sample app included in the project?